test/integration/job/OWNERS

```diff
@@ -6,5 +6,6 @@ approvers:
 reviewers:
   - sig-apps-reviewers
   - mimowo
+  - tenzen-y
```

Comment:
Should I postpone the one to another opportunity?

Reply:
I prefer the way it is - if you review the code then also the integration tests :) 

I would also prefer to give you a reviewer for the e2e tests, but these live in the specific apps/job.go along with other tests for apps, so we would need to hack the ownership per file.

Reply:
> I prefer the way it is - if you review the code, then also the integration tests :)

Yeah, I can take responsibility since I have experience with Job integration test implemention and review previously :)